support-frontend/assets/pages/contributions-landing/components/ContributionSurvey/ContributionsSurvey.jsx

```diff
@@ -35,8 +36,10 @@ export default function ContributionsSurvey(props: PropTypes) {
       </p>
       <AnchorButton
         href={surveyLink}
+        target="_blank"
```

Comment:
It might be worth adding a rel="noopener" or rel="noreferrer" to this - I think there's a slight security concern if you don't - as outlined [here](https://web.dev/external-anchors-use-rel-noopener/).

Reply:
nice! 👍